packages/react/src/ButtonGroup/ButtonGroup.tsx

```diff
@@ -73,24 +76,39 @@ const StyledButtonGroup = toggleStyledComponent(
   `,
 )
 
-export type ButtonGroupProps = ComponentProps<typeof StyledButtonGroup>
+export type ButtonGroupProps = ComponentProps<typeof StyledButtonGroup> & {
+  /**
+   * Settings to apply to the Focus Zone on the ButtonGroup container component. This is only used when role="toolbar" is supplied.
+   */
+  focusZoneSettings?: Partial<FocusZoneHookSettings>
```

Comment:
Would there be a way to capture relevant settings people might do as props on the component, instead? Just to avoid coupling this to `focusZoneSettings` if we want to refactor down the line.

Reply:
We could potentially forgo the `focusZoneSettings` prop since this is well scoped for only applying if `role="toolbar"` is true. Since the expectations are clear on how `role="toolbar"` should operate with keyboard, I think it's okay if consumers aren't able to pass their own settings.

I'm thinking if they truly needed to utilize their own settings, they could just apply their own `useFocusZone` outside of the component, as it _should_ replace the one that already exists, but I haven't tested so I'm not sure if it's true 😅

Reply:
Did a quick test and Tyler's right, `useFocusZone` in outer component will override behavior, removed `focusZoneSettings` here [e8782e2](https://github.com/primer/react/pull/5200/commits/e8782e2267161836db8029160ba455157cdcae36). Thanks for the suggestion both!